x-pack/plugins/enterprise_search/public/applications/app_search/components/search_ui/components/search_ui_graphic.scss

```
@@ -0,0 +1,188 @@
+.search-ui-graphic {
+  transform-style: preserve-3d;
+  transform: rotate3d(0, 1, 0, -8deg);
+
+  #search-area {
+    .outer-box {
+      fill:  $euiColorLightShade;
+    }
+    .field {
+      fill: $euiColorEmptyShade;
+    }
+    .search-icon {
+      fill: $euiColorDarkShade;
+    }
+    .type {
+      fill: $euiColorDarkShade;
+    }
+  }
+  #background {
+    fill: $euiColorLightestShade;
+  }
+  #results {
+    .outer-box {
+      fill: $euiColorEmptyShade;
+      stroke: $euiColorLightShade;
+      stroke-width: 1px;
```

optional, but since we're already (awesomely) using a bunch of EUI vars:
```suggestion
      stroke-width: $euiBorderWidthThin;
```

++ repeat this comment for all other instances of stroke `1px` in this file

---

Can do, but I wasn't sure how valuable it actually is to do for 1px values.

---

Prob not a huge deal but I kinda like it for readability. It makes sense in my head that the border goes from thin->thick on active state 😄

---

Ah yeah, great point.